### PR TITLE
Graphql basics

### PR DESCRIPTION
**Pull request by ramazanix**

### CODE REVIEW

**Comment by ramazanix** [comment]
@DmitryBarskov @KilinMikhail

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -29,12 +40,23 @@ def set_comment
...
+  end
+
   def comment_params
     params.require(:comment)
       .permit(:content, :task_id)
       .merge(
         {
-          user: current_user
+          user: current_user,
+          task: @task
```
Comment (DmitryBarskov) [inline app/controllers/comments_controller.rb:59]: At the same time you accept `task_id` and infer `task`. Could you remove some of them?

File: app/views/task_mailer/task_created.text.slim [hunk 2]
```diff
@@ -0,0 +1 @@
+| New product #{@task.name} created. You is a member of this product
```
Comment (DmitryBarskov) [inline app/views/task_mailer/task_created.text.slim]: ???

File: app/views/task_mailer/task_created.html.slim [hunk 3]
```diff
@@ -0,0 +1,8 @@
+h1
+  | New product #{@task.name} created.
```
Comment (DmitryBarskov) [inline app/views/task_mailer/task_created.html.slim]: what is product?

**Review by DmitryBarskov — CHANGES_REQUESTED** [review]

**Review by DmitryBarskov — APPROVED** [review]

**Pull request closed without merging**